tower.yml

```diff
@@ -0,0 +1,5 @@
+reports:
+  multiqc_report.html:
+    display: "MultiQC HTML report"
+  samplesheet.csv:
```

Comment:
I think this should be `samplesheet.valid.csv` and not `samplesheet.csv`

Reply:
Why do you think so? From my understanding this should be the samplesheet provided by the user, which is in our case `samplesheet.csv`. (Other pipelines (e.g. hicar) also have here `samplesheet.csv`.)

Reply:
My understanding is that only files that are published can be reported using tower.yml. And when I tested funcscan locally, I noticed that it only published samplesheet.valid.csv and not samplesheet.csv.

Reply:
Here is the relevant section in the tower documentation https://help.tower.nf/23.1/reports/overview/?h=tower.yml#path-pattern

Reply:
That is actually true. @louperelo since you use tower regularly – what is reported there, only the multiqc html?
Then we should either rename the output of `samplesheet_check.nf` or just update the file name in `tower.yml`.

Reply:
So, @louperelo and I just tried to check the Tower reports but since it seems unreachable atm, we'll wait until next week. 
Most likely, we should just remove the samplesheet entry in `tower.yml` because `samplesheet.csv` differs from `samplesheet.valid.csv` (since the [`check_samplesheet.py`](https://github.com/nf-core/funcscan/blob/dev/bin/check_samplesheet.py#L166) script adds an unnecessary column):

`samplesheet.csv`:
| sample | fasta |
| --- | --- |
| sample_1 | https://raw.githubusercontent.com/nf-core/test-datasets/funcscan/wastewater_metagenome_contigs_1.fasta.gz |
| sample_2 | https://raw.githubusercontent.com/nf-core/test-datasets/funcscan/wastewater_metagenome_contigs_2.fasta.gz |

`samplesheet.valid.csv`:
| sample | single_end | fasta |
| --- | --- | --- |
| sample_1 | | https://raw.githubusercontent.com/nf-core/test-datasets/funcscan/wastewater_metagenome_contigs_1.fasta.gz |
| sample_2 | | https://raw.githubusercontent.com/nf-core/test-datasets/funcscan/wastewater_metagenome_contigs_2.fasta.gz |

Reply:
Plus: the user already submits the samplesheet.csv, therefore not really necessary for him to receive it again in the output folder.

Reply:
I would argue to leave it as it is. We've never touched the samplesheet.csv validation form input checks, so this is whatever is in the template. If there is an issue here with tower it should be evalated to `nf-core/tools`.
 
(IMO)